.github/workflows/unittest.yml

```diff
@@ -0,0 +1,43 @@
+name: Helm Unittests
```

Comment:
Probably want to separate Unit Tests into two words for clarity (here and elsewhere)